### PR TITLE
Ttt logic

### PR DESCRIPTION
**Pull request by edward-vera**
idk if I'm doing this right =/

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -105,4 +90,52 @@ const resetBoard = () => {
...
+  } else { console.log('else')
+    changeMarker()
+  }
+}
+
+const horizontalWin = () => {
+  // Your code here to check for horizontal wins
+    if((board[0][0] == "X" && board[0][1] == "X" && board[0][2] == "X")
+    || (board[0][0] == "O" && board[0][1] == "O" && board[0][2] == "O")
+    ){return true}
```
Comment (juan-rod) [inline scripts.js:108]: lets put the return on a new line and then the closing } on it's own line too.
`if() {`
` return true`
`}`

**Review by juan-rod — APPROVED** [review]
Looks good! you can fix that one comment i left and then merge.

File: scripts.js (hunk 1 above)
Comment (edward-vera) [reply scripts.js:108]: putting it on it's own line as 
if() {
return true
}
resulted in it not working for some reason but doing it without the if() worked

**Comment by edward-vera** [comment]
Hey Juan!

I lost track of time yesterday and forgot to do the list filtering! I will
have it turned in by the end of the day!

Thanks,
Edward

**Pull request merged**